### ISSUE TITLE
Adding Entities is not obvious

### ISSUE THREAD

**Issue by whyfrycek** [huridocs/uwazi#820]
To add a new entity, now we need to know that the plus button in the upload section is used for that. Maybe we can find a more obvious way?

Reason: I had found it before and wanted to demo @bertver a feature, but could not find it immediatedly, and only after looking more carefully.

**Comment by danicatalan** [comment]
@whyfrycek absolutely! We are going to work on how to create multiple entities at once, and one of the things we are going to change is that button.

Closed in favour of #821, keep the talk in that issue :)

**Comment by danicatalan** [comment]
Reopened this issue because 821 has changed to [discussion], improved adding a label, still thinking on better solutions.

**Comment by danicatalan** [comment]
![screen shot 2017-02-27 at 10 31 47](https://cloud.githubusercontent.com/assets/6364153/23355871/f8bfba7a-fcd7-11e6-9442-7c56f3133502.png)

**Comment by danicatalan** [comment]
I've been thinking in this option and listed all the benefits of this new design:

- Buttons rendered as in Settings, that way _My files_ seems more like our current private sections.
- Buttons labels are more specific, so the user knows exactly which actions he's going to perform.
- Remove drag and drop box in favor of creating a specific Upload button (but I think we need to keep that functionality, so if a user drags a file inside the window, the file is uploaded.
- Top bar with _My files_ title, so the user always knows where he is (we need to use this top bar inside Settings too).

![screen shot 2017-02-27 at 13 43 57](https://cloud.githubusercontent.com/assets/6364153/23361849/fc27feb4-fcf2-11e6-8fda-6330f73d9b6e.png)

**Comment by txau** [comment]
@danicatalan much better. Maybe we can add a line "Drag your files in this area for uploading" at the bottom of the file list?

**Comment by danicatalan** [comment]
@txau It's unusable if there's a lot of files on the list because we don't have the Load more button on My Files.

If we want to keep the Drag and Drop box with the upload button we can change the text inside the box to remove the button, doesn't look bad and it's still usable, or maybe put the box as first item inside the list, both options are easy to implement:

![screen shot 2017-02-27 at 15 06 43](https://cloud.githubusercontent.com/assets/6364153/23364262/a22c98f0-fcfe-11e6-8111-5e243f8a9727.png)

![screen shot 2017-02-27 at 15 01 41](https://cloud.githubusercontent.com/assets/6364153/23364279/c1b4754e-fcfe-11e6-9d5c-7d00fce30628.png)

**Comment by txau** [comment]
@danicatalan I'm not talking abut keeping the same box, but making the whole area drag&drop (the file list). This is something some users have already complained as they tried to drag their files into the file list, not the box.

That way is not unusable, once users know the area is drag&drop ready, its ok for the message to be in the bottom.

**Comment by danicatalan** [comment]
@txau understand, is what I've proposed above.

Can we create another task to remove that Upload box and keep this just to modify the button?

**Comment by txau** [comment]
@danicatalan sure. Go for it please =)

**Comment by danicatalan** [comment]
The new CSS are working on [820-uploads-button](https://github.com/huridocs/uwazi/commits/820-uploads-button), we just need to add the functionality to those buttons.

The mobile version was overcrowded because the long texts, so I cut them. I hope they are still understandable.

![screen shot 2017-02-28 at 14 26 42](https://cloud.githubusercontent.com/assets/6364153/23406999/fe09d1a6-fdc1-11e6-9ad7-9228a374435e.png)

**Comment by konzz** [comment]
To be done in #887
